### PR TITLE
Proofread

### PR DESCRIPTION
**Pull request by Leybl**

### CODE REVIEW

**Comment by PRMerger12** [comment]
@Leybl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger16** [comment]
@Leybl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/kusto/api/rest/authentication.md [hunk 1]
```diff
@@ -11,36 +11,36 @@ ms.date: 10/30/2019
 ---
 # Authentication over HTTPS
 
-When using HTTPS the service supports the standard HTTP `Authorization` header
+When using HTTPS, the service supports the standard HTTP `Authorization` header
 for performing authentication.
 
-The supported HTTP authentication methods are:
+The HTTP authentication methods are:
```
Comment (orspod) [inline data-explorer/kusto/api/rest/authentication.md:17]:
```suggestion
The supported HTTP authentication methods are:
```

Comment (orspod) [reply data-explorer/kusto/api/rest/authentication.md:17]: Why did you take out supported?

File: data-explorer/kusto/api/rest/authentication.md [hunk 2]
```diff
@@ -11,36 +11,36 @@ ms.date: 10/30/2019
...
 
-* **Azure Active Directory**, through the `bearer` method.
+* **Azure Active Directory**, via the `bearer` method.
 
-When authenticating using Azure Active Directory, the `Authorization` header has
+When authenticating using the Azure Active Directory, the `Authorization` header has
```
Comment (orspod) [inline data-explorer/kusto/api/rest/authentication.md:21]:
```suggestion
When authenticating using Azure AD, the `Authorization` header has
```

File: data-explorer/kusto/api/rest/authentication.md [hunk 3]
```diff
@@ -11,36 +11,36 @@ ms.date: 10/30/2019
...
 the format:
 
 ```txt
 Authorization: bearer TOKEN
 ```
 
 Where `TOKEN` is the access token that the caller acquires by communicating with
-the Azure Active Directory service, with the following properties:
+the Azure Active Directory service. THe token has the following properties:
```
Comment (orspod) [inline data-explorer/kusto/api/rest/authentication.md:29]:
```suggestion
the Azure AD service. The token has the following properties:
```

File: data-explorer/kusto/api/rest/authentication.md [hunk 4]
```diff
@@ -11,36 +11,36 @@ ms.date: 10/30/2019
...
 * The Azure Active Directory service endpoint is
-  `https://login.microsoftonline.com/TENANT/`.
+  `https://login.microsoftonline.com/TENANT/`
 
-Where `TENANT` is the Azure Active Directory tenant ID or name. For example,
-services created under the Microsoft tenant can use
-`https://login.microsoftonline.com/microsoft.com/`. Alternatively, for user
-authentication only, the request can be made to
-`https://login.microsoftonline.com/common/` instead.
+Where `TENANT` is the Azure Active Directory tenant ID or name.
```
Comment (orspod) [inline data-explorer/kusto/api/rest/authentication.md:35]:
```suggestion
Where `TENANT` is the Azure AD tenant ID or name. 
```

File: data-explorer/kusto/api/rest/authentication.md [hunk 5]
```diff
@@ -11,36 +11,36 @@ ms.date: 10/30/2019
...
+Where `TENANT` is the Azure Active Directory tenant ID or name. 
+For example, services that are created under the Microsoft tenant can use
+`https://login.microsoftonline.com/microsoft.com/`. 
+Alternatively, for user authentication only, the request can be made to
+`https://login.microsoftonline.com/common/`.
 
 > [!NOTE]
-> The Azure Active Directory service endpoint changes when running in national clouds.
-> To change the endpoint that will be used, set an environment variable `AadAuthorityUri` to the required URI.
+> The Azure Active Directory service endpoint changes when it runs in national clouds.
```
Comment (orspod) [inline data-explorer/kusto/api/rest/authentication.md:42]:
```suggestion
> The Azure AD service endpoint changes when it runs in national clouds.
```

File: data-explorer/kusto/api/rest/authentication.md [hunk 6]
```diff
@@ -11,36 +11,36 @@ ms.date: 10/30/2019
...
+> To change the endpoint, set an environment variable `AadAuthorityUri` to the required URI.
 
 For more informaton, see the [authentication overview](../../management/access-control/index.md)
-and [guide to Azure Active Directory authentication](../../management/access-control/how-to-authenticate-with-aad.md).
\ No newline at end of file
+and the [guide to Azure Active Directory authentication](../../management/access-control/how-to-authenticate-with-aad.md).
```
Comment (orspod) [inline data-explorer/kusto/api/rest/authentication.md:46]:
```suggestion
and the [guide to Azure AD authentication](../../management/access-control/how-to-authenticate-with-aad.md).
```

**Comment by PRMerger14** [comment]
@Leybl : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/kusto/api/rest/authentication.md [hunk 7]
```diff
@@ -11,36 +11,36 @@ ms.date: 10/30/2019
...
+When using HTTPS, the service supports the standard HTTP `Authorization` header
 for performing authentication.
 
 The supported HTTP authentication methods are:
 
-* **Azure Active Directory**, through the `bearer` method.
+* **Azure Active Directory**, via the `bearer` method.
 
-When authenticating using Azure Active Directory, the `Authorization` header has
+When authenticating using the Azure AD, the `Authorization` header has
```
Comment (orspod) [inline data-explorer/kusto/api/rest/authentication.md:21]:
```suggestion
When authenticating using Azure AD, the `Authorization` header has
```

**Comment by PRMerger7** [comment]
@orspod : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**